subsys/bluetooth/host/gatt.c

```diff
@@ -4607,45 +4640,49 @@ int bt_gatt_unsubscribe(struct bt_conn *conn,
 		return -EINVAL;
 	}
 
+	if (!sub_has_subscription(sub, params)) {
+		return -EINVAL;
+	}
+
+	if (!sub_has_other_subscriptions_to_same_attribute(sub, params)) {
+		int gatt_write_err;
+
+		params->value = 0x0000;
```

Comment:
Suggest to create a 
```
#define BT_GATT_CCC_UNSUB			0x0000
```
and use that here

Reply:
I chose not to do this because it seems to me that `BT_GATT_CCC_*` is intended for `bt_gatt_subscribe_params.value`. And it makes little sense to allow BT_GATT_CCC_UNSUB there.